### PR TITLE
Add information about Json dependency

### PR DESCRIPTION
**Pull request by jvanbaarsen**
We removed the default dependecy on Poison, and made it optional. Add
something in the installation guide that tells users to install either
jason or posion.

### CODE REVIEW

File: source/elixir/installation.html.md [hunk 1]
```diff
@@ -24,11 +24,16 @@ Before you can compile the AppSignal gem make sure the build/compilation tools a
 -> **Note**: If you're running an umbrella application, see also our [umbrella installation guide](/elixir/installation/umbrella.html).
 
 1. Start by adding `appsignal` to your list of dependencies in `mix.exs`.
+   AppSignal requires you to depend on Json encoder. You can pick between
```
Comment (tombruijn) [inline source/elixir/installation.html.md]:
```suggestion
   AppSignal requires you to depend on a JSON encoder. You can pick between
```

**Comment by jeffkreeftmeijer** [comment]
This change will be included in the next minor release of the Elixir integration.

**Comment by jvanbaarsen** [comment]
@jeffkreeftmeijer When will that be? Do we have a tracking issue for that release somewhere so we can add this PR to that todolist?

**Comment by jvanbaarsen** [comment]
I've moved this to Waiting, since it is blocked by the release of the next release of the Elixir Integration

**Comment by tombruijn** [comment]
Let's publish then when 1.11 is released next week.

**Pull request merged**